modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectDefinitionSettingUtil.java

```diff
@@ -35,41 +38,89 @@ public class ObjectDefinitionSettingUtil {
 						objectDefinitionSettingLocalService.
 							createObjectDefinitionSetting(0L);
 
-				if (!StringUtil.equals(
+				if (StringUtil.equals(
+						ObjectDefinitionSettingConstants.
+							NAME_ACCEPTED_GROUP_IDS,
+						objectDefinitionSetting.getName()) ||
+					StringUtil.equals(
+						ObjectDefinitionSettingConstants.
+							NAME_ROOT_OBJECT_DEFINITION_IDS,
+						objectDefinitionSetting.getName())) {
+
+					return _setObjectDefinitionsSettingsByExternalReferenceCode(
+						serviceBuilderObjectDefinitionSetting,
+						objectDefinitionSetting);
+				}
+
+				if (StringUtil.equals(
 						ObjectDefinitionSettingConstants.
 							NAME_ACCEPTED_GROUP_EXTERNAL_REFERENCE_CODES,
 						objectDefinitionSetting.getName())) {
 
 					serviceBuilderObjectDefinitionSetting.setName(
-						objectDefinitionSetting.getName());
+						ObjectDefinitionSettingConstants.
+							NAME_ACCEPTED_GROUP_IDS);
+
+					String groupExternalReferenceCodes = String.valueOf(
+						objectDefinitionSetting.getValue());
+
 					serviceBuilderObjectDefinitionSetting.setValue(
-						String.valueOf(objectDefinitionSetting.getValue()));
+						StringUtil.merge(
+							TransformUtil.transform(
+								groupExternalReferenceCodes.split("\\s*,\\s*"),
+								groupERC -> {
+									Group group =
+										groupLocalService.
+											getGroupByExternalReferenceCode(
+												groupERC, companyId);
+
+									return String.valueOf(group.getGroupId());
+								},
+								String.class)));
 
 					return serviceBuilderObjectDefinitionSetting;
 				}
 
 				serviceBuilderObjectDefinitionSetting.setName(
-					ObjectDefinitionSettingConstants.NAME_ACCEPTED_GROUP_IDS);
+					ObjectDefinitionSettingConstants.
+						NAME_ROOT_OBJECT_DEFINITION_IDS);
 
-				String groupExternalReferenceCodes = String.valueOf(
-					objectDefinitionSetting.getValue());
+				String rootObjectDefinitionExternalReferenceCodes =
+					String.valueOf(objectDefinitionSetting.getValue());
 
 				serviceBuilderObjectDefinitionSetting.setValue(
 					StringUtil.merge(
 						TransformUtil.transform(
-							groupExternalReferenceCodes.split("\\s*,\\s*"),
-							groupERC -> {
-								Group group =
-									groupLocalService.
-										getGroupByExternalReferenceCode(
-											groupERC, companyId);
-
-								return String.valueOf(group.getGroupId());
+							rootObjectDefinitionExternalReferenceCodes.split(
+								"\\s*,\\s*"),
+							rootObjectDefinitionExternalReferenceCode -> {
+								ObjectDefinition objectDefinition =
+									objectDefinitionLocalService.
+										getObjectDefinitionByExternalReferenceCode(
+											rootObjectDefinitionExternalReferenceCodes,
+											companyId);
+
+								return String.valueOf(
+									objectDefinition.getObjectDefinitionId());
```

Comment:
@victorKammerer do the same thing as you did [here](https://github.com/liferay-bpm/liferay-portal/pull/4956/files#diff-72599e142fa29b706efb8cacd0edbd943e618f4347bc3a4950a309546caa71e9R354-R361), thanks!